src/pkg/bundle/common.go

```diff
@@ -212,9 +208,15 @@ func (b *Bundle) ValidateBundleResources(spinner *message.Spinner) error {
 	return nil
 }
 
-func getPkgPath(pkg types.Package, arch string) string {
+func getPkgPath(pkg types.Package, arch string, srcDir string) string {
 	var fullPkgName string
 	var path string
+	// Set path relative to the source directory if not absolute
+	if pkg.Path != "" {
```

Comment:
This works! Only thing left is to remove the outer `if pkg.Path != ""` statement. We check for that condition before this function is called